datacenter/dtr/2.2/guides/admin/configure/set-up-vulnerability-scans.md

```diff
@@ -52,6 +52,8 @@ Next, install the new license on the DTR instance.
 
 10. Navigate to where you saved the license key (`.lic`) file, select it, and click **Open**.
 
+![](../../images/security-scanning-setup-2.png)
```

Comment:
Can you add your images like `![](){: .with-border}` so that we add a css border around the images? That way they won't seem to float in the air.

[Check this](https://docs.docker.com/datacenter/ucp/2.1/guides/admin/configure/use-your-own-tls-certificates/) and look how the screenshots look a bit more clean and the page more organized 😄